lightbluetent/templates/home/register.html

```diff
@@ -11,7 +11,7 @@
         {%- if errors.first_name is defined %}
                 <small class="invalid-feedback">{{ errors.first_name }}</small>
         {%- endif %}
-        <small class="form-text text-muted">Please only use unaccented English characters in your name.</small>
+        <small class="form-text text-muted">Please use only the unaccented characters Aa - Zz.</small>
```

Comment:
Yeah, not 100% happy about this either. No technical restriction as far as I know. I originally took this directly from control-panel (https://github.com/SRCF/control-panel/blob/master/control/templates/signup/signup.html#L34) but something about the reference to the 'English' alphabet didn't sit right with me. Surveyed a few friends and was told that asking for 'Latin' characters might confuse people, hence I went with this for now. Thoughts?

If it's fine to use a wider range of characters then I suppose we can do that!

Reply:
That will probably have been because a socname in control panel will become a Unix username, and if you have anything too exotic in there it'll upset a lot of systems.  (We don't allow numbers for socs to avoid 99.9999% of possible CRSid collisions, since all but the very oldest of CRSids contain numbers.)  It'd also feed into the `SOCNAME.soc.srcf.net` domain, which could indeed be punycoded to allow non-English characters, although we don't have the mechanisms for that as it stands.

In conclusion, I think we can and should be more liberal here, since this value is will just feed into the 'path' part of some URL (which can get URL-encoded as necessary).